### DISCUSSION TITLE
Sitemap crawler?

### DISCUSSION THREAD

**Discussion by tharzzan** [FlowiseAI/Flowise#467] [Q&A]
Hey Henry, thanks a lot for creating this amazing project.

I wonder if Flowise can accept website sitemap url and automatically crawl every page mentioned there and ingest the data?

If not, is it something on a roadmap?

Thanks,
Andy

**Comment by HenryHengZJ** [comment]
@chungyau97 any thoughts?

**Comment by chungyau97** [comment]
@tharzzan definitely can create a function to extract all URL from the Website's Sitemap URL instead of web crawling method.

Will tie together with this PR #480

**Reply by tharzzan** [reply]
Yes please. Truly appreciate it. Thank you very much

**Reply by oswaldB** [reply]
Do you know when this PR will be merged?

Thank you

Very exciting feature!

**Comment by chungyau97** [comment]
#480 done and merged
